cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -465,53 +486,55 @@ def test_provision_with_boot_volume(request, instance_args, provider, soft_asser
 
     image = inst_args.get('template_name')
 
-    with provider.mgmt.with_volume(1, imageRef=provider.mgmt.get_template_id(image)) as volume:
-        # Set up automate
-        method = modified_request_class.methods.instantiate(name="openstack_CustomizeRequest")
+    volume = provider.mgmt.create_volume(1, imageRef=provider.mgmt.get_template(image).uuid)
+    request.addfinalizer(lambda: provider.mgmt.delete_volume(volume))
+
+    # Set up automate
+    method = modified_request_class.methods.instantiate(name="openstack_CustomizeRequest")
+    view = navigate_to(method, 'Details')
+    former_method_script = view.script.get_value()
+    with update(method):
+        method.script = dedent('''\
```

Comment:
This backslash is unnecessary within the dedent method call.

Reply:
Well it is and in same time it is not:
In [1]: from textwrap import dedent

In [2]: a=dedent("""\
   ...: a""")

In [3]: b=dedent("""
   ...: a""")

In [4]: a==b
Out[4]: False

In [5]: a
Out[5]: 'a'

In [6]: b
Out[6]: '\na'

The newline will certainly not break the ruby thingy, but ... we would have a newline on the start of the string.